tsb/cp/variables.tf

```diff
@@ -136,3 +136,7 @@ variable "cert-manager_enabled" {
 variable "ratelimit_enabled" {
   default = true
 }
+
+variable "id_propagation_enabled" {
```

Comment:
[nit] I'd suggest you rename this to `identity_propagation_enabled `, as `id` looks confusing to me. What I[nit] D? Could it refer to the request ID across hops? `identity_propagation_enabled` reads better and is more understandable.

Reply:
ack, fixed